src/devices/MAX31856/MAX31856.cs

```diff
@@ -0,0 +1,201 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.Spi;
+using UnitsNet;
+
+namespace Iot.Device.MAX31856
+{
+    /// <summary>
+    /// Add documentation here
+    /// </summary>
+    public class MAX31856 : IDisposable
+    {
+        private readonly byte _thermocoupleType;
+        private SpiDevice _spiDevice;
+
+        #region SpiSettings
+
+        /// <summary>
+        /// Spi Clock Frequency from the Technical Manual of the device
+        /// </summary>
+        public const int SpiClockFrequency = 5_000_000;
+
+        /// <summary>
+        /// Spi Clock Frequency from the Technical Manual of the device
+        /// </summary>
+        public const int DataFlow = 0;
+
+        /// <summary>
+        /// MAX31856 SPI Mode
+        /// </summary>
+        public const SpiMode SpiMode = System.Device.Spi.SpiMode.Mode1;
+
+        #endregion
+
+        /// <summary>
+        ///  Reads the temperature from the sensor
+        /// </summary>
+        /// <returns>
+        ///  Temperature in degrees celsius
+        /// </returns>
+        public Temperature GetTemperature() => Temperature.FromDegreesCelsius(ReadTemperature());
```

Comment:
As the ReadTemperature() function can raise exceptions, you should implement a 'TryGetTemperature' pattern. You'll find this pattern. See here: https://github.com/dotnet/iot/tree/master/src/devices#naming-of-functions

Reply:
Implemented, I'm not sure if I am error handling correctly. Also if it outputs bool I have have it read true if the device is working properly but I am not sure how to set the bool false if there is an exception